source/index.html.md.erb

```diff
@@ -4,7 +4,9 @@ title: The GDS Way
 
 # <%= current_page.data.title %>
 
-The GDS Way is guidance for how GDS teams should build their digital services. If you do not work for GDS and are looking for help creating and running digital services, please refer to the [Service Manual](https://www.gov.uk/service-manual).
+The GDS Way documents the specific technology, tools and processes that GDS teams use to build and operate the services GDS is responsible for. Like [GOV.UK Pay](https://www.gov.uk/government/publications/govuk-pay/govuk-pay), [GOV.UK Notify](https://www.gov.uk/government/publications/govuk-notify/govuk-notify) and [GOV.UK Verify (Identity Assurance Programme)](https://www.gov.uk/government/publications/introducing-govuk-verify/introducing-govuk-verify).
```

Comment:
I'd suggest removing the mention of Identity Assurance Program, I very rarely see it mentioned these days.

Reply:
Also I'd suggest using the product pages for these services, you can find them linked from the service-toolkit https://www.gov.uk/service-toolkit

Reply:
Thanks @davbo I have made a couple of changes.